src/components/VideoPlayer/VideoPlayerControls/VolumeButton/index.js

```diff
@@ -42,15 +42,20 @@ function VolumeButton({style, small}) {
         setSliderHeight(e.nativeEvent.layout.height);
     };
 
+    const gestureEventHandler = (event) => {
+        const val = Math.floor((1 - event.y / sliderHeight) * 100) / 100;
```

Comment:
**suggestion:** Extract some utility functions to `NumberUtils` for better semantics:

- `roundToTwoDecimalPlaces` or `asPercentage`
- `clamp`

Reply:
I very much agree